rel/overlay/etc/default.ini

```diff
@@ -23,6 +23,7 @@ file_compression = snappy
 ; and/or more OS page cache hits, but they can also increase overall response
 ; time for writes when there are many attachment write requests in parallel.
 attachment_stream_buffer_size = 4096
+db_definitions = [couch_dbs, mem3_dbs, couch_replicator_dbs, cassim_dbs]
```

Comment:
I want to mention that I do work on something bigger than the scope of this PR ([see](https://github.com/iilyak/plugerl)). However it wouldn't be one-click install either.